package-lock.json

```diff
@@ -18,16 +18,19 @@
         "@types/react-plotly.js": "^2.6.3",
         "@types/react-text-mask": "^5.4.14",
         "colord": "^2.9.3",
+        "konva": "^9.3.6",
```

Comment:
I'm curious why we have a package lock file again now in the repository and indeed why it changed here on this PR even though the package.json didn't change. When pnpm was introduced the package lock was removed and replaced by pnpm lock file. https://github.com/IPG-Automotive-UK/react-ui/pull/811

Maybe you can track down with @syedsalehinipg what has happened.

Reply:
I suspect maybe it was your figure PR that reintroduced it. Maybe you ran npm install rather than pnpm install?

Reply:
Probably was an `npm i` that caused this. `package-lock.json` removed